chanzuckerberg/single-cell-explorer
# feat: Add survey banner to Explorer

<details open>
  <summary>Argus Stack Details</summary>
  <br />
  <table>
    <tr>
      <th>Name</th>
      <td>rainandbare-7314-add-banner</td>
    </tr>
    <tr>
      <th>Short Name</th>
      <td>valid-elk</td>
    </tr>
    <tr>
      <th>Base URL</th>
      <td><a href="https://valid-elk.dev-sc.dev.czi.team" target="_blank">https://valid-elk.dev-sc.dev.czi.team</a></td>
    </tr>
  </table>
</details>


---

rdev --> https://valid-elk.dev-sc.dev.czi.team/d/super-cool-spatial.cxg/?spatial=true

## Reason for Change

- [#7314 ](https://github.com/chanzuckerberg/single-cell-data-portal/issues/7314)

## Changes

- add BottomBanner with new Airtable Survey link to Explorer
- uses the same local storage key as the data portal banner
- ensured that Breadcrumbs are still visible with banner present

### Bottom Banner Visible and Clickable
<img width="1428" alt="Screenshot 2024-08-02 at 12 43 18 PM" src="https://github.com/user-attachments/assets/5feb52d0-f3a2-457d-bcd4-2facfbf8c39b">

## Testing steps

- Check Explorer and see bottom banner 
- Click survey link to take you to correct airtable survey
- Click x button to dismiss banner - it will not return for 30 days after you do this (except in incognito browser)
- Check bottom banner on mobile screens as will as large. 
- Check that graph resizes correctly after you close the banner

## Checklist 🛎️
